.github/workflows/release.yml

```diff
@@ -41,15 +39,21 @@ jobs:
       - name: Lint
         run: npm run lint
 
+      - name: Compile
+        run: npm run build
+
       - name: Test
         run: npm test
         env:
           XATA_DATABASE_URL: ${{ secrets.INTEGRATION_TEST_DATABASE_URL }}
           XATA_API_KEY: ${{ secrets.INTEGRATION_TEST_API_KEY }}
           XATA_DATABASE_BRANCH: ${{ secrets.INTEGRATION_TEST_BRANCH }}
 
-      - name: Bump version
-        run: npx lerna version ${{ github.event.inputs.bump }} --yes
-
-      - name: Publish build
-        run: npx lerna publish from-package --no-verify-access --yes
+      - name: Create Release Pull Request or Publish to npm
```

Comment:
Maybe we can be more specific here? We are not creating a PR, right?

Reply:
It actually does one or the other depending on the branch.

If it's being called from feature branches, it creates/updates the release tracking PR.

If it's being called from the main branch, it publishes to npm the new release and cleans up the old changesets files.

Reply:
Makes sense 👍 !